Firestore/core/src/firebase/firestore/auth/token.h

```diff
@@ -56,9 +58,19 @@ class Token {
     return user_;
   }
 
+  bool is_valid() const {
+    return is_valid_;
+  }
+
+  /** Returns an invalid token. */
```

Comment:
Please add a porting note since this concept of an invalid token doesn't exist elsewhere. Something to indicate why we need it, etc.

Reply:
done